src/cloudant/__init__.py

```diff
@@ -24,30 +21,69 @@
 
 from .account import Cloudant, CouchDB
 
-
 @contextlib.contextmanager
 def cloudant(user, passwd, **kwargs):
     """
-    _cloudant_
-
-    Context helper to create a cloudant session and
+    Context helper to create a Cloudant session and
```

Comment:
Should this be called a "Context manager" for consistency with the docs for `@contextlib.contextmanager`? I'm not sure how useful that information is for users to understand what is supported here, but it seems strange to call it a helper when the decorator refers to a manager.